Gemfile

```diff
@@ -3,7 +3,7 @@
 source 'https://rubygems.org'
 
 group :development, :test do
-  gem 'gem_generator', '~> 0.3.0'
+  gem 'gem_generator', '~> 0.3'
```

Comment:
To be honest, it's a dangerous change, because any `0.x` version is unstable and should be locked further, on patch (the third) version.

But, hooray, now Faraday v2.9.0, dropped the support of non-secure Ruby 2.7 version. I'll change it later.